### PR TITLE
Artists db

### PR DESCRIPTION
**Pull request by JasonPKnoll**
What I Did:
- Add Artist DataBase:
    - Models folder and ruby file
- Add Artwork DataBase:
    - Artwork folder and ruby file
- Added Controllers:
    - Artists: Index, Create, Show, Update, Destroy methods
    - Artworks: Index,  Create, Show, Update, Destroy methods
    - Artist_Artworks:  Index method
    - EasterEgg: Index method
- Added Views
    - Artists:
      - edit.html.erb
      - index.html.erb
      - new.html.erb
      - show.html.erb
    - Artworks:
      - edit.html.erb
      - index.html.erb
      - new.html.erb
      - show.html.erb
    - Artist_Artworks
      - index.html.erb
    - EasterEgg
      - index.html.erb
- Updated Routes For
    - Artist - Index, ID (Show), New, Update, Delete
    - Artwork - Index, ID (Show), New, Update, Delete Roadblocks
    - Artist_artworks - Index

Roadblocks:
- All were worked past

 Testing:
- Artists:
    - Index: Story 1
    - Show: Story 2
- Artworks:
    - Index: Story 3
    - Show: Story 4
- Artists_Artworks
    - Index: Story 5 

Notes:
- Orderly gem added On

### CODE REVIEW

File: app/controllers/artist_artworks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class ArtistArtworksController < ApplicationController
+  def index
+    artist = Artist.find(params[:id])
```
Comment (kabcouwer) [inline app/controllers/artist_artworks_controller.rb:3]: interesting - I'd love to know what this does!

File: app/controllers/eastereggs_controller.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class EastereggsController < ApplicationController
+  def index
+  end
+end
```
Comment (kabcouwer) [inline app/controllers/eastereggs_controller.rb:4]: was this from that lesson lol - might not need it for the project but idk

File: app/views/eastereggs/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<center><h1>EASTER EGG</h1></center>
+
+<center><iframe width="420" height="315"
+src="https://www.youtube.com/embed/gMUEFZXkmDA">
+</iframe></center>
```
Comment (kabcouwer) [inline app/views/eastereggs/index.html.erb:5]: I'd bet my life on this being a rick roll

**Comment by kabcouwer** [comment]
Looks great Jason! Code looks really clean, tests look great, and I love how you already have the edit and add features for both databases - heck yea!

File: app/controllers/eastereggs_controller.rb (hunk 2 above)
Comment (JasonPKnoll) [reply app/controllers/eastereggs_controller.rb:4]: Haha, yes it is not needed for the project. I just decided to add it into the project when we were doing it for the session.

File: app/views/eastereggs/index.html.erb (hunk 3 above)
Comment (JasonPKnoll) [reply app/views/eastereggs/index.html.erb:5]: Really good guess, and it is not beneth me to do so, but alas I choose a different video/song

**Pull request merged**